### ISSUE TITLE
 Aarch64 Compatible Varnish  Binaries

### ISSUE THREAD

**Issue by odidev** [varnishcache/varnish-cache#3337]
Hi @Dridi,

I require varnish binaries for adding aarch64 support in docker-varnish (https://github.com/varnish/docker-varnish) and currently, through circleci aarch64 support has been added here. But as we can check, AArch64 binaries are not available at https://packagecloud.io/varnishcache/varnish64/. May I know, when can we have release of aarch64 binaries?

**Comment by Dridi** [comment]
Hi @odidev,

Github issues are only for actionable bugs, as stated in the issue template you get when you create a new issue.

Please refer to http://varnish-cache.org/support/index.html as stated in the issue template. For general discussion you might want to prefer the misc mailing list.

The answer is probably going to be that we aren't planning to provide official packages for aarch64 but maybe @gquintard will have a more nuanced answer? I'm sure he would notice your question on the misc list and reply there.

**Comment by gquintard** [comment]
@odidev, there's currently a bug with the ci, the debian `dsc` files are not properly generated. So that need to happen first if you want to help.
Then the bugwash needs to approve the weeklies on Mondays

**Comment by martin-g** [comment]
The nightly workflow is broken since two days.
I've just commented at https://github.com/varnishcache/varnish-cache/commit/df3994c3a2e19f36498c4f71f889fee6bc075776
